frontend/app/services/cardService.ts

```diff
@@ -1,38 +1,43 @@
+import { API_BASE_URL } from '../../env';
 import { CardData } from '../types/card';
 
 // Constants
-const API_DELAY = 500; // Simulated API delay in milliseconds
 export const CARDS_PER_PAGE = 20;
-
-// Helper function to generate mock card data
-const generateMockCard = (id: number): CardData => ({
-  id,
-  title: `Card ${id}`,
-  content: `This is the content of card ${id}. You can add more information here.`,
-});
-
-// Helper function to simulate API delay
-const simulateApiDelay = () =>
-  new Promise((resolve) => setTimeout(resolve, API_DELAY));
+const BASE_URL = API_BASE_URL;
```

Comment:
we don't need the rename here, you can just call it IDIOMS_BACKEND_URL or something like that